production/environment.yml

```diff
@@ -0,0 +1,33 @@
+name: openfe
```

Comment:
This seems a bit dangerous, how do we ensure that we're sufficiently updating this over time?

Reply:
CI will fail if we update deps in our main env and don't propagate them to this env. We need some way to control the difference for the docker env (like pinning python for example) than our main one. I could do it as overrides here: https://github.com/OpenFreeEnergy/openfe/pull/321/files#diff-5e76cf20a13dd71f36241a27c6ed523b01d79454c738cc64ea1490062e37385fR15 (for example I add `git`) but that seems as brittle as using this env, thoughts? Fundamentally, we have different environment requirements for our docker image, so we need to specify those somehow.

Reply:
I guess if CI fails then it's good enough for now. We could just have an action that reads in the yaml files and checks the version ranges are compatible, but it's not for today.